apps/web/components/eventtype/CreateEventType.tsx

```diff
@@ -200,16 +200,43 @@ export default function CreateEventTypeButton(props: Props) {
             )}
             <TextField label={t("title")} placeholder={t("quick_chat")} {...register("title")} />
 
-            <TextField
-              label={t("url")}
-              required
-              addOnLeading={
-                <InputLeading>
-                  {process.env.NEXT_PUBLIC_WEBSITE_URL}/{pageSlug}/
-                </InputLeading>
-              }
-              {...register("slug")}
-            />
+            <div className="hidden sm:block">
+              <TextField
+                label={t("url")}
+                required
+                addOnLeading={
+                  <InputLeading>
+                    {process.env.NEXT_PUBLIC_WEBSITE_URL}/{pageSlug}/
+                  </InputLeading>
+                }
+                {...register("slug")}
+              />
+            </div>
+
+            {process.env.NEXT_PUBLIC_WEBSITE_URL !== undefined &&
+            process.env.NEXT_PUBLIC_WEBSITE_URL?.length >= 21 ? (
+              <div className="block sm:hidden">
+                <TextField
+                  label={`${t("url")}: ${process.env.NEXT_PUBLIC_WEBSITE_URL}`}
+                  required
+                  addOnLeading={<InputLeading>/{pageSlug}/</InputLeading>}
+                  {...register("slug")}
+                />
+              </div>
+            ) : (
+              <div className="block sm:hidden">
+                <TextField
+                  label={t("url")}
+                  required
+                  addOnLeading={
+                    <InputLeading>
+                      {process.env.NEXT_PUBLIC_WEBSITE_URL}/{pageSlug}/
+                    </InputLeading>
+                  }
+                  {...register("slug")}
+                />
```

Comment:
Can we prevent registering multiple fields? (both desktop and mobile) Also I think we can apply the visual fixes more granularly.

Reply:
Simplified, no special responsive treatment.